cmd/documentation/exchanges_templates/subscription.tmpl

```diff
@@ -25,9 +25,21 @@ The template is provided with a single context structure:
 
 Subscriptions may fan out many channels for assets and pairs, to support exchanges which require individual subscriptions.  
 To allow the template to communicate how to handle its output it should use the provided directives:
-- AssetSeparator should be added at the end of each section related to assets
+- AssetSeparator should be added at the end of each section related to assets:
```

Comment:
`:` has been added and below was this intentional?

Reply:
Fixed b9fe4d4b

Reply:
thanks!